src/include/cluster/dialogs.rb

```diff
@@ -1049,37 +1067,38 @@ def ServiceDialog
       contents = VBox(
         VSpacing(1),
         Frame(
-          _("Booting"),
-          RadioButtonGroup(
-            Id("bootcorosync"),
-            HBox(
-              HSpacing(1),
-              VBox(
-                Left(
-                  RadioButton(
-                    Id("on"),
-                    Opt(:notify),
-                    _("On -- Start pacemaker during boot")
-                  )
-                ),
-                Left(
-                  RadioButton(
-                    Id("off"),
-                    Opt(:notify),
-                    _("Off -- Start pacemaker manually")
+          _("Cluster start at booting time enable/disable"),
```

Comment:
Any reason change the layout?

Reply:
The reason why is I think booting section stays the same with "cluster start/stop" section seems harmonious :)
And the enable/disable status also shows up seems friendly.